### DISCUSSION TITLE
Expand CORe50 support for Detection and Segmentation

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#416] [Feature Request]
[CORe50 originally supports Detection and Segmentation](https://vlomonaco.github.io/core50/). It would be nice to integrate this in Avalanche.

**Comment by pkraison** [comment]
It would nice to see this next

**Reply by vlomonaco** [reply]
great, vote the feature-request ⬆️ we will implement most voted features first!

**Reply by vlomonaco** [reply]
added issue #547 related to this feature-request.
